frontend/packages/dev-console/integration-tests/features/getting-started/getting_started_tour_dev_perspective

```diff
@@ -0,0 +1,77 @@
+
+
+@regression
+Scenario: Quick tour when user logs in for the first time
+   Given user is in logging page for the first time
+   When user loggings to console
+   And user switches to developer perspective if not already there
+   And the Getting Started experience is automatically displayed
+   And user can see first screen as 'Welcome to Dev perspective' 
+   And user can see the button labels on the first screen are 'Get Started' & 'Close'
+   And user clicks on Get started 
+   And user can see next screen appears on Perspective switcher
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Monitoring tab
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Search tab
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Command Line Terminal
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Help
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears is Final Screen saying 'You are ready to go!'
+   And user can see the button labels as 'Got it' & 'Back to tour'
+   And user can clicks on 'Got it' button
+   Then user is taken to Add view in developer perspective
+
+@regression
+Scenario: Quick tour from help menu
+   Given user is in developer perspective
+   When user opens help menu on top right
+   And user clicks on 'Guided Tour' option
+   And the Getting Started experience is automatically displayed
+   And user can see first screen as 'Welcome to Dev perspective' 
+   And user can see the button labels on the first screen are 'Get Started' & 'Close'
+   And user clicks on Get started 
+   And user can see next screen appears on Perspective switcher
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Monitoring tab
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Search tab
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Command Line Terminal
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears on Help
+   And user can see the button labels as 'Next' & 'Back'
+   And user can clicks on Next button
+   And user can see next screen appears is Final Screen saying 'You are ready to go!'
+   And user can see the button labels as 'Got it' & 'Back to tour'
+   And user can clicks on 'Got it' button
+   Then user is taken to Add view in developer perspective
```

Comment:
You have repeated everything from above scenario. Just make this scenario for Guided Tour from help menu.

Reply:
First of all, not everything is ditto i.e. the starting of it is from the help menu and it is a different scenario from help menu